tests/foreman/cli/test_contentview.py

```diff
@@ -3962,3 +3962,23 @@ def test_positive_inc_update_should_not_fail(self, module_org):
         assert result[2]
         content_view = ContentView.info({'id': content_view['id']})
         assert '1.1' in [cvv_['version'] for cvv_ in content_view['versions']]
+
+    @pytest.mark.tier1
+    def test_positive_pulp_workers_running_state(self, target_sat):
+        """Ensure that 6 pulpcore-workers are in active (running) state
+
+        :BZ: 1848111
+
+        :id: f0b8b354-1a30-41c3-93d7-fb51e99c82b6
+
+        :customerscenario: true
+
+        :expectedresults: 6 pulpcore-workers are in active (running) state
+
+        :SubComponent: Pulp
+        """
+        output = target_sat.execute('systemctl status pulpcore-worker* | grep Active')
+        result = output.stdout.split('\n')
```

Comment:
If you use `rstrip()` you won't have to deal with the empty string element in the list.